src/com/mambu/apisdk/services/OrganizationService.java

```diff
@@ -134,6 +137,30 @@ public Currency getCurrency() throws MambuApiException {
 	}
 
 	/**
+	 * Requests a currency for a supplied currency code.
+	 * 
+	 * @param currencyCode
+	 *            the currency code for the currency to be searched in Mambu. Must not be null.
+	 * 
+	 * @return Currency having as currency code the currency code passed as parameter to this method call. In case a
+	 *         wrong currency code (there is no currency having currency code having this code) is passed to this method

```

Comment:
Maybe we could replace "there is no currency having currency code having this code" with something like just "there is no currency having this code".